components/Layout/Layout.module.css

```diff
@@ -28,6 +28,43 @@
   overflow: hidden;
 }
 
+.backgroundimage {
+    position: relatives;
+    width: 100%;
+    height: 555px;
+}
+
+.backgroundtextleft {
+position: absolute;
```

Comment:
du könntest noch Prettier in VSCode installieren, damit deine Dateien automatisch richtig formatiert werden -> https://www.alphr.com/use-prettier-vs-code/